NeoSwift/Util/Base58.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  Base58.swift
+//  NeoSwift
+//
+//  Created by Luís Silva on 11/09/17.
+//  Copyright © 2017 drei. All rights reserved.
+//
```

Comment:
How heavily influenced is this by the previous library? If it is no trivial than we should provide some attribution to Swift-Base-X author.

Reply:
It's not. I based myself on C/C++ base58 algorithms.